WEB_SERVICE/WEB/_LOW/DNS_INAPPROPRIATE_TTL_VALUES/description.md

```diff
@@ -0,0 +1,7 @@
+Time-to-Live (TTL) values in DNS records define how long a DNS resolver should cache the record before querying the authoritative DNS server again. Incorrectly setting TTL values, whether too high or too low, can lead to inefficiencies in DNS propagation, which can impact website performance, DNS load balancing, and service failover.
+
+Setting the right TTL values is essential for balancing performance and DNS efficiency. Inappropriate TTL values can cause various issues, depending on whether they are set too high, too low, or not set at all. Each of these scenarios can have negative consequences:
```

Comment:
highlight more explicitly how this affects DNS propagation, specifically in the context of changes to DNS records..

Reply:
it's mentioned here "- **High TTL values (e.g., greater than 86400 seconds)**: Reduce query traffic and enhance performance, but can delay important DNS updates (e.g., IP address changes), potentially causing downtime or outdated records being served.  "